src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Logging/ProjectOutputWindowPaneProvider.cs

```diff
@@ -52,13 +52,10 @@ public ProjectOutputWindowPaneProvider(IProjectThreadingService threadingService
         private static IVsOutputWindowPane CreateProjectOutputWindowPane(IVsOutputWindow outputWindow)
         {
             Guid paneGuid = s_projectOutputWindowPaneGuid;
-            HResult hr = outputWindow.CreatePane(ref paneGuid, VSResources.OutputWindowPaneTitle, fInitVisible: 1, fClearWithSolution: 0);
-            if (hr.Failed)
-                throw hr.Exception;
 
-            hr = outputWindow.GetPane(ref paneGuid, out IVsOutputWindowPane pane);
-            if (hr.Failed)
-                throw hr.Exception;
+            Verify.HResult(outputWindow.CreatePane(ref paneGuid, VSResources.OutputWindowPaneTitle, fInitVisible: 1, fClearWithSolution: 0));
```

Comment:
I used to deliberately use the previous pattern so that the exceptional path showed up in code coverage. Undecided about this...

Reply:
You could make the same argument about all other validation we have, such as `Requires.NotNull`.

Reply:
Requires is different, it's a contract break, it will never happen in a normal code path

I'm okay with this.

Reply:
True. I can see the desire to track places we're not explicitly testing what happens for erroneous HResults. It's one commit if we want to back it out in future.